cf-net/cf-net.c

```diff
@@ -740,9 +768,62 @@ static int CFNetGet(ARG_UNUSED CFNetOptions *opts, const char *hostname, char **
         return -1;
     }
 
-    int ret = CFNetGetFile(hostname, remote_file, local_file);
+    CFNetThreadData **threads = (CFNetThreadData**) xcalloc(n_threads, sizeof(CFNetThreadData*));
+    for (int i = 0; i < n_threads; i++)
+    {
+        threads[i] = (CFNetThreadData*) xcalloc(1, sizeof(CFNetThreadData));
+        threads[i]->data = (GetFileData*) xcalloc(1, sizeof(GetFileData));
```

Comment:
no casting is needed nor recommended in all of the above, in C void* casting is implicit.